awpy/types.py

```diff
@@ -676,6 +676,9 @@ class RoundStatistics(TypedDict):
 # Type to represent tile id for navigation tiles.
 TileId: TypeAlias = int
 
+# Type to represent player position (list of floats [x, y, z])
+PlayerPosition: TypeAlias = list[float]
+
```

Comment:
Should we make these into tuple[float, float, float] in the future? I think this wouldnt necessarily be a point for this MR as i think its still lists in other parts of awpy. But what do you think in general?

Reply:
Sorry not sure what MR means in this context. I however did make the change to update to `tuple[float, float, float]` since the length of the list is known so I think a tuple makes sense here.

Reply:
Ah I see MR might've been Merge Request - my bad.

I think I know what you mean now. I just ran 'pyright' and saw there were a few errors with PlayerPosition in other parts of the code, especially with using `find_closest_area` in `map_control.py` and the point argument in the function.

Yes I agree - I think this should be updated to tuple[float, float, float] in a future MR; however, I'll keep it as list[float] for now due to the resulting issues caused by changing to tuple. I'll make an issue to track this update.

Reply:
Yeah, sorry. MR is what gitlab calls PRs.